zerver/lib/outgoing_http.py

```diff
@@ -21,7 +21,7 @@ class OutgoingHTTPAdapter(requests.adapters.HTTPAdapter):
     def __init__(self, role: str, timeout: int, *args: Any, **kwargs: Any) -> None:
         self.role = role
         self.timeout = timeout
-        super().__init__(*args, *kwargs)
+        super().__init__(*args, **kwargs)
```

Comment:
@alexmv FYI -- is this a change we should backport?

Reply:
Yikes -- thanks for the catch!  This is reasonable but not urgent to backport -- there are no callsites that pass anything that'd end up in `kwargs`.